assets/helpers/internationalisation/country.js

```diff
@@ -89,11 +145,70 @@ function fromString(s: string): ?IsoCountry {
     case 'uk': return 'GB';
```

Comment:
How about `if(s.toLowerCase() === 'uk'){
return 'GB';
} else {
return s.toUpperCase();
}`
?

Reply:
I tried to do that, the problem there is Flow. Flow doesn't consider `s.toUpperCase` to be a valid `isoCountry`, it thinks it is a string and therefore fails.

Reply:
I don't think we should let Flow bully us, just disable it for that line

Reply:
Yeap, maybe you are right. I personally, prefer the fact that flow hard checks this.

Reply:
This problem was addressed here: https://github.com/guardian/support-frontend/pull/504 Specifically here: 30da8ff4aefdd2a6a583ff33da1163d93e23f40f